errors.go

```diff
@@ -168,3 +168,5 @@ const (
 	ErrWrongSchemaVaersion           = 109 // Wrong schema version, current: %d, in request: %u
 	ErrSlabAllocMax                  = 110 // Failed to allocate %u bytes for tuple in the slab allocator: tuple is too large. Check 'slab_alloc_maximal' configuration option.
 )
+
+var ErrStrangerConn = fmt.Errorf("the passed connected request doesn't belong to the current connection or connection pool")
```

Comment:
Do we really need this in the public API or is there another way?
```suggestion
var errStrangerConn = fmt.Errorf("the passed connected request doesn't belong to the current connection or connection pool")
```